flutter/analysis_options.yaml

```diff
@@ -24,3 +24,4 @@ linter:
   rules:
     prefer_relative_imports: true
     unnecessary_brace_in_string_interps: true
+    implementation_imports: true
```

Comment:
currently, we're using `package:pedantic`, is there a more aggressive solution? often I see us enabling new rules that should be avoided in the 1st place :D

Reply:
As per https://dart-lang.github.io/linter/lints/ there are three official rule sets: pedantic, effective_dart and flutter. 
There's also https://pub.dev/packages/lint which describes itself as very strict.

Reply:
thanks, I'll have a look at this one (lint) later this week.